lib/watch/interface.go

```diff
@@ -30,6 +31,12 @@ type Interface interface {
 	ResultChan() <-chan Event
 }
 
+// WatcherConverter represents a formatter for calico resources returned by Watch.
+type WatcherConverter interface {
```

Comment:
I don't think this belongs here.  We aren't adding conversion to general watcher interface - just the interface clientv3 watcher.

Reply:
Fair enough. I'll move them around.